Update CONTRIBUTING.md with a proper Splunk CLA link (#110)
signalfx/gdi-specification: specification/templates/CONTRIBUTING.md

See the [LICENSE](LICENSE) file for our repository's licensing. We will ask you to
confirm the licensing of your contribution.

## CLA

> All contributors must execute the [Splunk Contributor License Agree (CLA)](https://www.splunk.com/en_us/form/contributions.html) form.

Reviewer: ```suggestion

All contributors must sign off the [Splunk Contributor License Agreement (CLA)](https://www.splunk.com/en_us/form/contributions.html) form.
```

See the [LICENSE](LICENSE) file for our repository's licensing. We will ask you to

> confirm the licensing of your contribution.

Reviewer: Could we remove `We will ask you to confirm the licensing of your contribution.` if one already has to sign the CLA?

> ## CLA

Reviewer 2: Is a new section needed? The contributor license agreement seems appropriate to be included in the `Licensing` section itself.

```suggestion
```

If not, can we make it a sub-section of `Licensing` and not use an acronym here?

```suggestion
### Contributor License Agreement
```

Author: Good point, made it a subsection.

> All contributors must sign the [Splunk Contributor License Agreement (CLA)](https://www.splunk.com/en_us/form/contributions.html).

Reviewer: I just checked and the whole file is written in "2nd form".
Therefore it would be better to rephrase it to:

```suggestion
Before contributing, you must sign the [Splunk Contributor License Agreement (CLA)](https://www.splunk.com/en_us/form/contributions.html).
```

Also maybe it would be better just to put it as a bullet in the `Contributing via Pull Requests` section?
